specifyweb/frontend/js_src/lib/components/FormCells/FormTable.tsx

```diff
@@ -169,6 +171,16 @@ export function FormTable<SCHEMA extends AnySchema>({
     mode,
   });
 
+  const validExpandedViewDefinition = viewToChange.has(viewName)
+    ? useViewDefinition({
```

Comment:
you can't call hooks conditionally
how many times??? 😁

Reply:
:grimacing: so sorry